mysql-test/mroonga/storage/column/int/t/with_index_null_value.test

```diff
@@ -0,0 +1,42 @@
+# Copyright(C) 2024 Takuya Kodama <otegami@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+--source ../../../../include/mroonga/have_mroonga.inc
+--source ../../../include/mroonga/load_mroonga_functions.inc
+
+--disable_warnings
+DROP TABLE IF EXISTS items;
+--enable_warnings
+
+CREATE TABLE items (
+  name VARCHAR(255),
+  price INT NULL,
+  KEY price_index(price)
+) DEFAULT CHARSET=utf8mb4;
+
+INSERT INTO items VALUES ("hamburger", 200);
+INSERT INTO items VALUES ("smile", 0);
+INSERT INTO items VALUES ("sample chips", NULL);
+
+SELECT mroonga_command("index_column_diff --table items#price_index --name index");
+
+SELECT * FROM items WHERE price = 0;
+
+SELECT * FROM items WHERE price >= 100;
```

Comment:
This is needless.

Reply:
fix: [800d984](https://github.com/mroonga/mroonga/pull/790/commits/800d9847ee518059275e7418ae73e44cabfab006)